distribution/docs/src/main/resources/_contents/_endpoints/csw-endpoint-contents.adoc

```diff
@@ -0,0 +1,2118 @@
+
+==== CSW Endpoint
+
+The CSW endpoint enables a client to search collections of descriptive information (metadata) about geospatial data and services.
+
+===== Installing CSW Endpoint
+
+The CSW Endpoint is installed by default with a standard installation in the ${ddf-spatial} application.
+
+===== Configuring CSW Endpoint
+
+The CSW endpoint has no configurable properties.
+It can only be installed or uninstalled.
+
+===== CSW Endpoint URL
+
+The CSW endpoint is accessible from ${secure_url}/services/csw.
+
+====== CSW Endpoint Operations
+
+`GetCapabilities` Operation:: The `GetCapabilites` operation is meant to describe the operations the catalog supports and the URLs used to access those operations. The CSW endpoint supports both `HTTP GET` and `HTTP POST` requests for the `GetCapabilities` operation. The response to either request will always be a `csw:Capabilities` XML document. This XML document is defined by the http://schemas.opengis.net/csw/2.0.2/CSW-discovery.xsd[CSW-Discovery XML Schema].
+
+.`GetCapabilities` HTTP GET
+The `HTTP GET` form of `GetCapabilities` uses query parameters via the following URL:
+
+.`GetCapabilities` KVP Encoding
+----
+${secure_url}/services/csw?service=CSW&version=2.0.2&request=GetCapabilities
```

Comment:
Since we're using the csw version in quite a few places, should we consider using a `${}` variable so it's not too painful to update when next time it changes?